nucliadb_models/nucliadb_models/vectors.py

```diff
@@ -32,3 +38,31 @@ class VectorSet(BaseModel):
 
 class VectorSets(BaseModel):
     vectorsets: Dict[str, VectorSet]
+
+
+# Resource rendering
+
+
+class GetUserVector(BaseModel):
+    vector: List[float]
+    labels: Optional[List[str]] = None
+    start: int
+    end: int
+
+
+class UserVectors(BaseModel):
+    vectors: Optional[Dict[str, GetUserVector]]
+
+
+class UserVectorSet(BaseModel):
+    vectors: Optional[Dict[str, UserVectors]]
```

Comment:
Can we simplify the nested dicts here? if not it's ok, it just feels very complex

Reply:
I do not understand the proposal